modules/rostests/apitests/ntdll/CMakeLists.txt

```diff
@@ -139,7 +162,8 @@ set_target_properties(ntdll_apitest
     ENABLE_EXPORTS TRUE
     DEFINE_SYMBOL "")
 
-target_link_libraries(ntdll_apitest rtl_test_lib wine uuid ${PSEH_LIB})
+target_compile_definitions(ntdll_apitest PRIVATE NTDLL_APITEST)
```

Comment:
`NTDLL_APITEST` does not exist (yet)...
https://github.com/search?q=repo%3Areactos%2Freactos%20NTDLL_APITEST&type=code

Reply:
Removed.